.github/workflows/maven-verify-test.yml

```diff
@@ -32,3 +32,4 @@ jobs:
       ff-goal: test
       verify-goal: clean install
```

Comment:
Do we really need install? Isn't verify sufficient and saves time?

Reply:
it is only test workflow executed in this repo - show what happens when `ff-goal` and `verify-goal` have different commands